notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -4,7 +4,7 @@
       "cell_type": "code",
```

Comment:
<p>This tutorial uses the&nbsp;<a href="https://www.tensorflow.org/datasets/catalog/tf_flowers" target="_blank">flower dataset</a>&nbsp;provided by TensorFlow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>